src/ota_metadata/legacy2/metadata.py

```diff
@@ -163,7 +164,7 @@ def _prepare_metadata(
                     dst=_metadata_jwt_fpath,
                 )
             ]
-            condition.wait()  # wait for download finished
+            condition.wait(cfg.DOWNLOAD_INACTIVE_TIMEOUT)  # wait for download finished
```

Comment:
```suggestion
            if not condition.wait(cfg.DOWNLOAD_INACTIVE_TIMEOUT):  # wait for download finished
                    raise TimeoutError("Download of metadata.jwt timed out")
```

Reply:
I think add `timeout` to `condition.wait` is unintended. 
Here it means that the download for each file MUST be finished within 5mins, so it will unintentionally fail the downloading if the network is just very slow.

I think we should still set the timeout logic as when network is fully disconnected and there is not network traffic for `DOWNLOAD_INACTIVE_TIMEOUT`.